routers/user/auth_openid.go

```diff
@@ -467,7 +467,7 @@ func RegisterOpenIDPost(ctx *context.Context) {
 
 	// Send confirmation email, no need for social account.
 	if setting.Service.RegisterEmailConfirm && u.ID > 1 {
-		mailer.SendActivateAccountMail(ctx.Locale, u)
+		mailer.SendActivateAccountMail(ctx.Locale.Language(), u)
```

Comment:
at the moment the lang from context is used ... not sure if we should just switch to DefaultLang